introduction/architectural-overview.md

```diff
@@ -1 +1,35 @@
 # Architectural overview
+
+## Tiers
+Valtimo is a based on a three-tier architecture. 
+
+-	The presentation tier is the user interface, based on Angular. It’s referred to as Frontend. 
+-	The (business) logic tier is the heart of Valtimo, built in Kotlin/Java. It’s referred to as Backend.
+-	For the data tier PostgreSQL is used – MySQL is supported for versions < 9.
```

Comment:
MySQL is also supported for Valtimo 9 & 10 & 11.

We have no plans to drop support for MySQL databases

Reply:
changed.